pkg/cmd/eventlistener/describe.go

```diff
@@ -277,3 +293,22 @@ func isBindingSpecExist(bindings []*v1alpha1.EventListenerBinding) bool {
 	}
 	return specExist
 }
+
+func isBindingNameExist(bindings []*v1alpha1.EventListenerBinding) bool {
+	nameExist := false
+	for _, j := range bindings {
+		if j.Name != "" {
+			nameExist = true
+		}
+	}
+	return nameExist
+}
```

Comment:
Could you explain the need for this func? Is it possible to have bindings without names?

Reply:
Nevermind, I see the difference between ref/name.